api/envoy/extensions/filters/http/composite/v3/composite.proto

```diff
@@ -57,4 +58,12 @@ message ExecuteFilterAction {
   // Only one of ``typed_config`` or ``dynamic_config`` can be set.
   DynamicConfig dynamic_config = 2
       [(udpa.annotations.field_migrate).oneof_promotion = "config_type"];
+
+  // Percentage of traffic the action filter will be skipped. By default this is 0.
+  // This is to support sampling behavior for certain filters like ext_proc, access_logging ...
+  // If a none-zero number is configured for ``skip_percent``, only a portion of traffic,
+  // i.e, 1 - skip_percent, needs to be processed by the filter.  In such case, a dice roll
+  // is done after a matching condition met, the action will only be taken if the dice roll
+  // returns true. Otherwise, the action filter is skipped.
+  type.v3.FractionalPercent skip_percent = 3;
```

Comment:
The reason to name this as skip_percent, not sampling_percent is for backward compatibility. i.e, by default with this field not configured, zero percent will be skipped, i.e,, 100% traffic will be processed by the action filter.

Reply:
I don't think we need to do that. Since this is a message field, we can use the absence of the field to enable 100% of traffic sampled, and then when set it will indicate the sample rate.

Reply:
I'm good with this API for the various reasons already articulated in discussions. I know @markdroth had some lingering concerns so I'll defer to him for LGTM.

Reply:
Thanks for the comments! Changed the proto field name to be "sample_percent".